src/client/script-tag-config.coffee

```diff
@@ -56,9 +56,9 @@ initParams = {
   embed: { default: true, transform: transform.bool },
   autoStart: { default: true, transform: transform.bool },
   accessToken: { default: null }
+  customerId: { default: null }
```

Comment:
I will not change this one for now... I do not want to add to much logic into this. It seems overkill.